tests/framework/validators/ValidatorTest.php

```
@@ -351,4 +351,42 @@ public function testScalarAttributeNames(): void
         $validator = SafeValidator::createValidator('safe', $model, [1]);
         $this->assertSame([1], $validator->getValidationAttributes(1));
     }
+
+    public function testInlineValidatorClosure(): void
+    {
+        $model = new DynamicModel(['attr' => 1]);
+        $val = new InlineValidator([
+            'method' => function ($attribute, $params, $validator, $current) {
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$params'. (undefined)

(UnusedFormalParameter)